book/src/hello_world.md

```diff
@@ -9,14 +9,14 @@ Let's write [a minimal VM](https://github.com/powdr-labs/powdr/blob/main/test_da
 Then let's generate a proof of execution for the valid prover input `0` (since for `0 + 1 - 1 == 0`)
 
 ```console
-powdr pil hello_world.asm --field bn254 --force --inputs 0 --prove-with halo2
```

Comment:
Maybe we should have these in a shell script which we run in tests, and include fragments here the same way we do for powdr code snippets.

Reply:
Can be another PR